src/openrct2/ride/coaster/meta/Hypercoaster.h

```diff
@@ -39,7 +39,7 @@ constexpr const RideTypeDescriptor HypercoasterRTD =
     SET_FIELD(RatingsCalculationFunction, ride_ratings_calculate_corkscrew_roller_coaster),
     SET_FIELD(RatingsMultipliers, { 50, 30, 10 }),
     SET_FIELD(UpkeepCosts, { 40, 20, 80, 11, 3, 10 }),
-    SET_FIELD(BuildCosts, { 105, 5, 50, }),
+    SET_FIELD(BuildCosts, { 62.5_GBP, 2.5_GBP, 50, }),
```

Comment:
Shouldn't it be 52.5_GBP?

Reply:
Definitely looks like a typo
```suggestion
    SET_FIELD(BuildCosts, { 52.5_GBP, 2.5_GBP, 50, }),
```